### ISSUE TITLE
HomeBrew error while updating

### ISSUE THREAD

**Issue by vcdx71** [unifi-poller/unifi-poller#328]
brew has been failing to upgrade from 2.0.1 to 2.1 for a while now, just getting around to reporting it.

```
❯ brew upgrade unifi-poller
==> Upgrading 1 outdated package:
golift/mugs/unifi-poller 2.0.1 -> 2.1.0
==> Upgrading golift/mugs/unifi-poller 2.0.1 -> 2.1.0 
==> Downloading https://golift.io/unifi-poller/archive/v2.1.0.tar.gz
Already downloaded: /Users/mike/Library/Caches/Homebrew/downloads/a4dfc75deb8467315d561379658dcceca63dd58b70c86ddf1c4a0cfcf5f6fea5--unifi-poller-2.1.0.tar.gz
==> make install VERSION=2.1.0 ITERATION=862 PREFIX=/usr/local/Cellar/unifi-poller/2.1.0 ETC=/usr/local/etc
Last 15 lines from /Users/mike/Library/Logs/Homebrew/unifi-poller/01.make:
                but was required as: gopkg.in/russross/blackfriday.v2
go install: version is required when current directory is not in a module
        Try 'go install github.com/davidnewhall/md2roff@latest' to install the latest version
make: *** [/Users/mike/Library/Caches/Homebrew/go_mod_cache/bin/md2roff] Error 1
make: *** Waiting for unfinished jobs....
[ -z "/usr/local/opt/upx/bin/upx" ] || /usr/local/opt/upx/bin/upx -q9 unifi-poller
                       Ultimate Packer for eXecutables
                          Copyright (C) 1996 - 2020
UPX 3.96        Markus Oberhumer, Laszlo Molnar & John Reiser   Jan 23rd 2020

        File size         Ratio      Format      Name
   --------------------   ------   -----------   -----------
  14268736 ->   5058576   35.45%   macho/amd64   unifi-poller

Packed 1 file.

If reporting this issue please do so at (not Homebrew/brew or Homebrew/core):
  https://github.com/golift/homebrew-mugs/issues
```

**Comment by davidnewhall** [comment]
I actually ran into this bug recently while building on Ubuntu. I think I solved it in this release: https://github.com/davidnewhall/md2roff/releases/tag/v0.0.1
And it was put into use here: https://github.com/davidnewhall/unpackerr/pull/120/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R9

All of the builds for macOS, docker and Linux worked, so this should be OK now. The next release should fix this. I'll try to get one out this week.
